flink-yarn/src/main/java/org/apache/flink/yarn/AbstractYarnClusterDescriptor.java

```diff
@@ -722,12 +697,30 @@ public ApplicationReport startAppMaster(JobGraph jobGraph, YarnClient yarnClient
 
 		// Setup jar for ApplicationMaster
 		LocalResource appMasterJar = Records.newRecord(LocalResource.class);
-		LocalResource flinkConf = Records.newRecord(LocalResource.class);
```

Comment:
nit: for me it would be easier to understand/review the changes done here, if there was a separate PR for changing formatting (splitting one param per line)